src/memory.c

```diff
@@ -168,7 +168,7 @@ real_read6502(uint16_t address, bool debugOn, uint8_t bank)
 		if (ramBank < num_ram_banks) {
 			return RAM[0xa000 + (ramBank << 13) + address - 0xa000];
 		} else {
-			return OPEN_BUS_READ;
+			return (address >> 8) & 0xff; // open bus read
```

Comment:
I think I would have split this into a separate commit to not conflate this unrelated change with the command line option change. Or is this needed because of the MHZ change?

Reply:
This project has never followed a strict practice needing a separate commit or PR for trivial changes.  I don't think it's worth splitting this one out, but I'll keep it in mind so reverts are easier.

Reply:
I agree the change seems trivial but I wasn't sure if it was  somehow related to the MHZ change somewhere down the line